javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,12 @@
+function ExampleConstructor() {
+}
+
+console.log('value of prototype:', Object.getPrototypeOf(ExampleConstructor));
+console.log('typeof prototype:', typeof Object.getPrototypeOf(ExampleConstructor));
+
+var afunction = new ExampleConstructor();
```

Comment:
Consider renaming. When we call a constructor function it creates an object. A variable name such as `anObject` might be a better indicator of the value it holds.